brh.data-commons.org/metadata/aggregate_config.json

```diff
@@ -43,8 +43,8 @@
       }
     },
     "JCOIN": {
-      "mds_url": "https://jcoin.datacommons.io/",
-      "commons_url": "jcoin.datacommons.io/",
+      "mds_url": "https://jcoin.datacommons.io",
```

Comment:
oh actually I think you should not change this

Reply:
Do you mean the `mds_url` @mfshao ?

Reply:
I matched whatever all the other `mds_url` was set to, so I think we should keep it like this.